frontend/src/views/MigrationHistoryDetail.vue

```diff
@@ -199,21 +235,42 @@ export default defineComponent({
   setup(props) {
     const store = useStore();
 
-    const migrationHistory = computed((): MigrationHistory => {
-      return store.getters["instance/migrationHistoryById"](
-        idFromSlug(props.migrationHistorySlug)
+    const database = computed((): Database => {
+      return store.getters["database/databaseById"](
+        idFromSlug(props.databaseSlug)
+      );
+    });
+
+    // get all migration histories before (include) the one of given id, ordered by descending version.
+    const prevMigrationHistoryList = computed((): MigrationHistory[] => {
+      const migrationHistoryList: MigrationHistory[] = store.getters[
+        "instance/migrationHistoryListByInstanceIdAndDatabaseName"
+      ](database.value.instance.id, database.value.name);
+      return migrationHistoryList.filter(
+        (mh) => mh.id <= idFromSlug(props.migrationHistorySlug)
       );
     });
 
+    const migrationHistory = computed(
+      (): MigrationHistory => prevMigrationHistoryList.value[0]
+    );
+
+    const lastRecordedSchema = computed(
```

Comment:
Also we should use consistent language language here, previous and last doesn't seem to be the proper works. Do we mean by current and previous?

Reply:
comment added